src/test/python/StatementsWithTransactions.py

```diff
@@ -0,0 +1,107 @@
+''' Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+'''
+
+import psycopg2 as pg
+import sys
+
+def create_connection(port):
+  try:
+    connection = pg.connect(user="postgres",
+                            database="postgres",
+                            host="localhost",
+                            port=port)
+    return connection
+  except Exception as e:
+    print(e)
+    return None
+
+def set_property(connection, property, value):
+  if property == 'autocommit':
+    connection.autocommit = bool(value)
+  elif property == 'deferrable':
+    connection.deferrable = bool(value)
+  elif property == 'isolation_level':
+    if value == '0':
+      connection.set_isolation_level(0)
+    else:
+      connection.isolation_level = int(value)
```

Comment:
nit: do we need this if-else?

Reply:
Yeah, because we aren't allowed to do `connection.isolation_level = 0` or `set_session(isolation_level = 0)`. The only way to set it to 0 is by using `set_isolation_level()` function

Reply:
Ah, makes sense. Thanks!